doc/ipython-notebooks/classification/HashedDocDotFeatures.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### by Evangelos Anagnostopoulos (GitHub ID: <a href=\"http://github.com/van51\">van51</a>).<br> Special thanks to my mentors for this project on GSoC 2013 : Soeren Sonnenburg, Olivier Chapelle and Benoit Rostykus</br>"
+    "#### By Evangelos Anagnostopoulos (GitHub ID: <a href=\"http://github.com/van51\">van51</a>)"
```

Comment:
@karlnapf  That's wasn't much useful information! Your call, if you say I can add that again ?
But this looks good :)